config.toml

```
@@ -66,6 +66,14 @@ weight = 1
 #time_format_default = "02.01.2006"
 #time_format_blog = "02.01.2006"
 
+[languages.bn]
+title = "ক্লাউড নেটিভ শব্দকোষ"
+description = "CNCF ক্লাউড নেটিভ শব্দকোষ প্রকল্পটি ক্লাউড নেটিভ অ্যাপ্লিকেশন সম্পর্কে কথা বলার সময় ব্যবহৃত সাধারণ পদগুলির জন্য একটি রেফারেন্স হিসাবে ব্যবহার করার উদ্দেশ্যে।"
+languageName = "বাংলা(Bengali)"
+contentDir = "content/bn”
```

```suggestion
contentDir = "content/bn"
```
" and ” are different characters. This will lead a build failure ;)

---

Updated that